fuzzing/README.md

```diff
@@ -0,0 +1,35 @@
+# Fuzzing
+
+Automated fuzzing tests for RIOT network applications.
+
+## Setup
+
+The following additional dependencies are required:
+
+* [afl][afl homepage]
+* [libasan][sanitizers github] (optional but recommended)
```

Comment:
Want to put actual URLs here? 😉

Reply:
Those are proper markdown links. Markdown supports two style of links inline and reference, those are reference links.